src/components/FlowRunTimeline.vue

```diff
@@ -52,6 +53,7 @@
           :sub-node-labels="subFlowRunLabels"
           :selected-node-id="selectedNode?.id"
           :expanded-sub-nodes="expandedSubFlowRuns"
+          @update:visible-date-range="updateVisibleDateRange"
```

Comment:
Rather than using the update event to emit the pattern we prefer to use is a get/set like this. As is you're mutating the prop and emitting the event. At the very least you need to change `v-model:visibleDateRange="..."` to `:visibleDateRange="..."`

```typescript
const visibleDateRange = computed({
  get() {
    return props.visibleDateRange
  },
  set(value) {
    emit('update:visibleDateRange', value)
  }
})
```

Reply:
Ah gotchya, ty!